sympy/physics/quantum/dagger.py

```diff
@@ -2,7 +2,7 @@
 
 from __future__ import print_function, division
```

Comment:
Can you also remove this line and the blank line below it?? Future imports are no longer needed.